new/server/app.js

```diff
@@ -42,20 +36,21 @@ app.use(
   })
 );
 
-//server initialization
-app.listen(3002, ()=>
-{
-  console.log('connected to 3002');
-})
+
 
 //connect to mongo database
 mongoose.connect(`mongodb+srv://sankha:sankha@cluster0-blgns.mongodb.net/test?retryWrites=true&w=majority`,{ useNewUrlParser:true, useUnifiedTopology:true })
 .then((res)=>
 {
     console.log('mongodb connected');
 })
```

Comment:
Why is the database an online link? Since we are still develping it, we need to have a local database set up. We can do thisafterwards.

Reply:
oh, that was by the previous developer. I'll change that right away.

Reply:
@thebishaldeb previous developer used MongoDB atlas for database as it was given in tutorial, so this link is his credentials he had set online. Its better if we keep this like this for now. I tried to take it to local but it showed me error. We can work on this on a later PR.

Reply:
As a suggestion, I think rather than making a local database, we can get credentials by the name of the project and use it here. Its more robust according to me 👍

Reply:
Yeah cool. Merging it.